lua/neocord/init.lua

```diff
@@ -838,12 +842,11 @@ function neocord:update_for_buffer(buffer, should_debounce)
         self.workspaces[project_path].updated_at = activity_set_at
         activity.timestamps = self.options.show_time == 1
             and {
-              start = self.workspaces[project_path].started_at,
+              start = self.workspaces[project_path].updated_at,
```

Comment:
My guess is that this line is what caused `global_timer` to work incorrectly. I needed to change this anyway since the timer would not restart after coming back from idle, but it took hours for me to trace why that was happening to this line. I don't see any other references to the `started_at` in this project, so I removed it a few lines below. Is this alright, or am I missing something?

In retrospect, should we allow the user to determine whether the timer should reset, or whether it should maintain the same time as before idling?

Reply:
> In retrospect, should we allow the user to determine whether the timer should reset, or whether it should maintain the same time as before idling?

Same time i think

Reply:
Or by default have it reset and make an option to allow for keeping time